<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="4" total="9" passed="4" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-04-03T19:08:24 CEST" name="Default suite" finished-at="2020-04-03T19:09:49 CEST" duration-ms="84412">
    <groups>
    </groups>
    <test started-at="2020-04-03T19:08:24 CEST" name="Default test" finished-at="2020-04-03T19:09:49 CEST" duration-ms="84412">
      <class name="tests.Summer_DressesTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.Summer_DressesTest@75329a49]" started-at="2020-04-03T19:08:24 CEST" name="setUp" finished-at="2020-04-03T19:08:35 CEST" duration-ms="10296" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="test01_clickFirst()[pri:0, instance:tests.Summer_DressesTest@75329a49]" started-at="2020-04-03T19:08:35 CEST" name="test01_clickFirst" finished-at="2020-04-03T19:08:43 CEST" duration-ms="8147" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test01_clickFirst -->
        <test-method signature="test02_Order()[pri:0, instance:tests.Summer_DressesTest@75329a49]" started-at="2020-04-03T19:08:43 CEST" name="test02_Order" finished-at="2020-04-03T19:08:47 CEST" duration-ms="4417" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test02_Order -->
        <test-method signature="test03_Values()[pri:0, instance:tests.Summer_DressesTest@75329a49]" started-at="2020-04-03T19:08:47 CEST" name="test03_Values" finished-at="2020-04-03T19:08:58 CEST" duration-ms="10399" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test03_Values -->
        <test-method signature="test04_registration()[pri:0, instance:tests.Summer_DressesTest@75329a49]" started-at="2020-04-03T19:08:58 CEST" name="test04_registration" finished-at="2020-04-03T19:09:40 CEST" duration-ms="42255" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test04_registration -->
        <test-method signature="test05_logInBack()[pri:0, instance:tests.Summer_DressesTest@75329a49]" started-at="2020-04-03T19:09:40 CEST" name="test05_logInBack" finished-at="2020-04-03T19:09:49 CEST" duration-ms="8827" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [2] but found [(empty)]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [2] but found [(empty)]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at tests.Summer_DressesTest.test05_logInBack(Summer_DressesTest.java:125)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test05_logInBack -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.Summer_DressesTest@75329a49]" started-at="2020-04-03T19:09:49 CEST" name="tearDown" finished-at="2020-04-03T19:09:49 CEST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- tests.Summer_DressesTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
